src/test/java/com/google/api/codegen/testsrc/artman_showcase.yaml

```diff
@@ -0,0 +1,34 @@
+common:
+  api_name: showcase
+  api_version: v1alpha1
+  organization_name: google
+  service_yaml: showcase.yaml
+  gapic_yaml: v1alpha1/showcase_gapic.yaml
+  src_proto_paths:
+  - v1alpha1
+  proto_deps:
+  - name: google-common-protos
+artifacts:
```

Comment:
I'm 80% sure we should be able to remove the whole `artifacts` config, since it isn't used anymore.